src/lib/model/InboundThirdpartyTransactionModel.js

```diff
@@ -0,0 +1,109 @@
+/**************************************************************************
+ *  (C) Copyright ModusBox Inc. 2019 - All rights reserved.               *
+ *                                                                        *
+ *  This file is made available under the terms of the license agreement  *
+ *  specified in the corresponding source code repository.                *
+ *                                                                        *
+ *  ORIGINAL AUTHOR:                                                      *
+ *       Sridhar Voruganti - sridhar.voruganti@modusbox.com               *
+ **************************************************************************/
+
+'use strict';
+
+const {
+    BackendRequests,
+    HTTPResponseError,
+} = require('@internal/requests');
+
+const {
+    MojaloopRequests,
+    Ilp,
+    Errors,
+} = require('@mojaloop/sdk-standard-components');
+
+const shared = require('@internal/shared');
+
+/**
+ *  Models the operations required for inbound third party transaction requests
+ */
+class InboundThirdpartyTransactionModel {
+    constructor(config) {
+        this._cache = config.cache;
+        this._logger = config.logger;
+        this._dfspId = config.dfspId;
+        this._expirySeconds = config.expirySeconds;
```

Comment:
where is this property used?

Reply:
Deleted, actually we will use same inbound and outbound models for 331, 332 stories, so I thought these properties might be required